source/content/guides/new-relic/02-activate-new-relic.md

```diff
@@ -49,28 +49,6 @@ New Relic&reg; is automatically enabled for all application containers added to
 
 In both cases, there is not a one-to-one mapping between Pantheon users and New Relic&reg; users. Every Pantheon user who selects **Go to New Relic** from the Pantheon dashboard is logged in as the same New Relic&reg; user.
 
-## Configure Ping Monitors for Availability
```

Comment:
This is being removed as it's a duplicate of the content in https://docs.pantheon.io/guides/new-relic/monitor-new-relic#configure-ping-monitors-for-availability